bindings/matrix-sdk-crypto-ffi/src/olm.udl

```diff
@@ -55,8 +50,8 @@ enum CryptoStoreError {
     "CryptoStore",
     "OlmError",
     "Serialization",
-    "Identifier",
```

Comment:
Hmm, what's this change about?

Reply:
Ah, this (and one other change in the Rust code that's also about variant order in an enum) is so the UDL and Rust definitions match exactly. That is required so redundant definition of the enum between proc-macro metadata and UDL is allowed.